### PR TITLE
Th3r3d patch 1

### PR DESCRIPTION
**Pull request by adulau**

### CODE REVIEW

File: GrayZone/machinetag.json [hunk 1]
```diff
@@ -1,51 +1,51 @@
 {
   "namespace": "GrayZone",
   "description": "Gray Zone of Active defense includes all elements which lay between reactive defense elements and offensive operations. It does fill the gray spot between them. Taxo may be used for active defense planning or modeling.",
-  "version": 2,
+  "version": 3,
```
Comment (th3r3d) [inline GrayZone/machinetag.json:4]: We will be releasing version 3 somewhere next year, it will be maybe better to number this verion as 2.1. Thank You

**Review by th3r3d — CHANGES_REQUESTED** [review]
Thank You for cleanup, You are awesome. COuld You pls just fix the version number, otherwise everything is cool. Thx again.

File: GrayZone/machinetag.json (hunk 1 above)
Comment (adulau) [reply GrayZone/machinetag.json:4]: Thanks a lot for the feedback. The only issue, it should be an integer. I can also put the latest date of update if this is fine for you.

Comment (th3r3d) [reply GrayZone/machinetag.json:4]: Yep, the date will be just fine.
Thank You

**Pull request merged**